app/views/smart_answers/_previous_answers.html.erb

```diff
@@ -1,10 +1,20 @@
 <% if @presenter.current_question_number > 1 %>
   <table class="govuk-table previous-answers-table">
     <caption class="govuk-table__caption">
-      <h2 class="govuk-heading-m">Previous answers</h2>
+      <% if local_assigns[:results_page] && @presenter.hide_previous_answers_on_results_page? %>
```

Comment:
Would it be possible to just pass in the variable to hide the previous answers? That way the partial doesn't need to know anything about which template its nested in

```suggestion
      <% if local_assigns[:hide_previous_answers] %>
```

Reply:
Thanks @theseanything - yeah, quite right - that looks much better.